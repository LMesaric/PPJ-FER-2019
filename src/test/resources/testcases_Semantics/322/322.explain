semantic: function f called with wrong number of params
note: 'f ( 2 )' on line 7
