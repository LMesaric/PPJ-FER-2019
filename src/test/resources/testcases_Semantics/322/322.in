<prijevodna_jedinica>
 <prijevodna_jedinica>
  <vanjska_deklaracija>
   <definicija_funkcije>
    <ime_tipa>
     <specifikator_tipa>
      KR_INT 1 int
    IDN 1 f
    L_ZAGRADA 1 (
    KR_VOID 1 void
    D_ZAGRADA 1 )
    <slozena_naredba>
     L_VIT_ZAGRADA 2 {
     <lista_naredbi>
      <naredba>
       <naredba_skoka>
        KR_RETURN 3 return
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       BROJ 3 2
        TOCKAZAREZ 3 ;
     D_VIT_ZAGRADA 4 }
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 5 int
   IDN 5 main
   L_ZAGRADA 5 (
   KR_VOID 5 void
   D_ZAGRADA 5 )
   <slozena_naredba>
    L_VIT_ZAGRADA 6 {
    <lista_naredbi>
     <lista_naredbi>
      <naredba>
       <izraz_naredba>
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <postfiks_izraz>
                       <primarni_izraz>
                        IDN 7 f
                      L_ZAGRADA 7 (
                      <lista_argumenata>
                       <izraz_pridruzivanja>
                        <log_ili_izraz>
                         <log_i_izraz>
                          <bin_ili_izraz>
                           <bin_xili_izraz>
                            <bin_i_izraz>
                             <jednakosni_izraz>
                              <odnosni_izraz>
                               <aditivni_izraz>
                                <multiplikativni_izraz>
                                 <cast_izraz>
                                  <unarni_izraz>
                                   <postfiks_izraz>
                                    <primarni_izraz>
                                     BROJ 7 2
                      D_ZAGRADA 7 )
        TOCKAZAREZ 7 ;
     <naredba>
      <naredba_skoka>
       KR_RETURN 8 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 8 0
       TOCKAZAREZ 8 ;
    D_VIT_ZAGRADA 9 }
