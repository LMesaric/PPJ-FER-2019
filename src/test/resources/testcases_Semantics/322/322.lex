KR_INT 1 int
IDN 1 f
L_ZAGRADA 1 (
KR_VOID 1 void
D_ZAGRADA 1 )
L_VIT_ZAGRADA 2 {
KR_RETURN 3 return
BROJ 3 2
TOCKAZAREZ 3 ;
D_VIT_ZAGRADA 4 }
KR_INT 5 int
IDN 5 main
L_ZAGRADA 5 (
KR_VOID 5 void
D_ZAGRADA 5 )
L_VIT_ZAGRADA 6 {
IDN 7 f
L_ZAGRADA 7 (
BROJ 7 2
D_ZAGRADA 7 )
TOCKAZAREZ 7 ;
KR_RETURN 8 return
BROJ 8 0
TOCKAZAREZ 8 ;
D_VIT_ZAGRADA 9 }
