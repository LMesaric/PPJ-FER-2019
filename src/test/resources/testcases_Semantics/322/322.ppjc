int f(void)
{
  return 2;
}
int main(void)
{
  f(2);
  return 0;
}