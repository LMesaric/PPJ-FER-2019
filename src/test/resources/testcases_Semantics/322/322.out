<postfiks_izraz> ::= <postfiks_izraz> L_ZAGRADA(7,() <lista_argumenata> D_ZAGRADA(7,))
